test/get_previous_releases.py

```diff
@@ -108,6 +108,10 @@ def download_binary(tag, args) -> int:
     platform = args.platform
     if tag < "v23" and platform in ["x86_64-apple-darwin", "arm64-apple-darwin"]:
         platform = "osx64"
+    elif tag == "v23.0" and platform == "arm64-apple-darwin":
```

Comment:
does this not affect 24 and later as well?

Reply:
Yes, it already affects the current v24.0.1 arm64 tarball. But that is not yet being pulled into the `get_previous_releases.py` list and I figured it's possible/likely(?) that a subsequent v24.x release will have the proper signing before any v24 is part of this backwards compatibility list.

Reply:
See also https://github.com/bitcoin/bitcoin/pull/26586

Once there is a pull with a fix, it can set the tag here. But I am not aware of a pull/plan yet, so it seems best to avoid assuming it.